testbench.py

```diff
@@ -0,0 +1,54 @@
+from concurrent.futures import ThreadPoolExecutor
+
+
+def add_numbers(a: int, b: int) -> int:
+    print(f"[ADD_NUMBERS] Starting with parameters: a={a}, b={b}")
+    result = a + b
+    print(f"[ADD_NUMBERS] Returning result: {result}")
+    return result
+
+
+def test_threadpool() -> None:
```

Comment:
please integrate this test bench functionality into the tracer e2e test we already have